CONTRIBUTING.md

```diff
@@ -67,6 +67,20 @@ go get golang.org/x/tools/cmd/vet
 
 For more information on `go vet`, [read the GoDoc](https://godoc.org/golang.org/x/tools/cmd/vet).
 
+Revision Control Systems
+------
+
+Go has the ability to import remote packages via revision control systems with the `go get` command.  To ensure that you can retrieve any remote package, be sure to install the following rcs software to your system.
+Currently the project only depends on `git` and `mercurial`.
```

Comment:
We only depend on `git` now. References to all other source-control systems can be removed.